services/gcn_service/gcn_service.py

```diff
@@ -64,7 +66,8 @@ def service():
             for message in consumer.consume():
                 payload = message.value()
                 consumer.commit(message)
-                user_id = 1
+                if payload.find(b'Broker: Unknown topic or partition') != -1:
```

Comment:
Btw this is because when a Kafka topic hasn't been pushing notices recently, it says it can't be accessed. However we never caught that, and the payload was still passed to the `post_gcnevent_from_xml` causing errors. The errors themselves aren't a big problem, but we did end up with a lot of ugly prints in the logs.